rclcpp/include/rclcpp/executor.hpp

```diff
@@ -560,6 +608,46 @@ class Executor
   virtual void
   spin_once_impl(std::chrono::nanoseconds timeout);
 
+private:
```

Comment:
Not sure whether this shall be in the private section - will change it if it's done differently.

Reply:
I prefer `protected` so that it can be messed with by a derived class